packages/contracts/src/manager/libraries/LibDiamond.sol

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IDiamondCut } from "../interfaces/IDiamondCut.sol";
+import { IDiamondLoupe } from "../interfaces/IDiamondLoupe.sol";
+import "@openzeppelin/contracts/interfaces/IERC165.sol";
+import { IERC173 } from "../interfaces/IERC173.sol";
+
+// Remember to add the loupe functions from DiamondLoupeFacet to the diamond.
+// The loupe functions are required by the EIP2535 Diamonds standard
+
+error InitializationFunctionReverted(address _initializationContractAddress, bytes _calldata);
+
+library LibDiamond {
```

Comment:
All dependencies should be added through `forge install` and not copy paste contracts unless we are forking them and plan to maintain them ourselves.

Reply:
> All dependencies should be added through `forge install` and not copy paste contracts unless we are forking them and plan to maintain them ourselves.

Explained above